### DISCUSSION TITLE
Using JsPsych for Data Collection - SONA / MTurk

### DISCUSSION THREAD

**Discussion by BobRoss048** [jspsych/jsPsych#1142] [Q&A]
Hello, 

First off - I apologize if this isn't necessarily a Q/A question.  If it's not, let me know and I'll fix it!  

Secondly, I've completed my experiment (thanks to a lot of help from here) and now want to use it for data collection.  As such, I have a question.  Is it possible to use my experiment (in an HTML / Javascript / CSS format) with systems such as SONA & have it collect data for me?  

Currently, it does show data at the end of the experiment, but is it possible for it to store that data & keep track of it for multiple participants, so that I can run an analysis on the many people who have taken my experiment?  I know it's possible to send it to other people, but if I were to use it on SONA / Mturk, could I set it up in such a way to store that data for me over the duration of the experiment?  

Thank you all again for all your help!

**Comment by jodeleeuw** [comment]
Hi @BobRoss048 ,

Typically SONA / MTurk would handle the recruitment of subjects, and a web server would be used to handle data storage. The easiest approach to data storage right now is probably to use [cognition.run](https://cognition.run). You could then have SONA / MTurk direct your participants to the URL provided by cognition.run.
